plugins/polio/tasks/vaccine_authorizations_mail_alerts.py

```diff
@@ -3,13 +3,16 @@
 from datetime import timedelta
 
 from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
 from django.core.mail import send_mail
 from django.shortcuts import get_object_or_404
 
 from hat import settings
 from iaso.models import Team
 from plugins.polio.models import VaccineAuthorization
 
+vaccine_dashboard_link = f"{Site.objects.get_current()}/dashboard/polio/vaccinemodule/nopv2authorisation/accountId/1/order/-current_expiration_date/pageSize/20/page/1"
```

Comment:
Do we hard code the account id everywhere in polio? if not maybe get it from the user?

It will work in prod, but if we get multiple testing environments, this may not work for future testers.

Reply:
I changed it